invisible_cities/cities/icaro.py

```diff
@@ -0,0 +1,532 @@
+"""
+-----------------------------------------------------------------------
+                                Icaro
+-----------------------------------------------------------------------
+
+
+"""
+
+import numpy as np
+import pandas as pd
+import tables as tb
+
+from .. reco                import tbl_functions as tbl
+from .. reco.corrections    import ASectorMap
+from .. reco.corrections    import maps_coefficient_getter
+from .. reco.corrections    import correct_geometry_
+from .. reco.corrections    import norm_strategy
+from .. reco.corrections    import get_normalization_factor
+from .. reco.corrections    import apply_all_correction
+from .. io.krmaps_io        import write_krmap
+from .. io.krmaps_io        import write_krevol
+from .. io.krmaps_io        import write_mapinfo
+from .. core                import fit_functions as fitf
+from .. core.core_functions import in_range
+from .. core.core_functions import shift_to_bin_centers
+
+from .. dataflow          import dataflow as fl
+
+from .. types.ic_types    import Measurement
+from .. types.ic_types    import MasksContainer
+
+from .  components import city
+from .  components import print_numberofevents
+from .  components import dst_from_files
+from .  components import quality_check
+from .  components import kr_selection
+from .  components import map_builder
```

Comment:
`map_builder` and `add_krevol` are defined in `icaro.py` and not in components (I followed the convention that if a function is used only in the module it stays in the module itself)